app.py

```diff
@@ -154,6 +155,10 @@ def graph_refresh_solver() -> None:
                     indexes=[index_url],
                     output=_SOLVER_OUTPUT,
                     transitive=False,
+                    ceph_bucket_prefix=os.environ["THOTH_CEPH_BUCKET_PREFIX"],
+                    ceph_bucket_name=os.environ["THOTH_CEPH_BUCKET"],
+                    ceph_host=urlparse(os.environ["THOTH_S3_ENDPOINT_URL"]).netloc,
+                    deployment_name=os.environ["THOTH_DEPLOYMENT_NAME"]
```

Comment:
As for thoth-common. IMHO this could go directly to thoth-common as its application configuration.